nextcord/interactions/application.py

```diff
@@ -0,0 +1,177 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING, List, Optional, Tuple, Union
+
+from .base import Interaction
+
+__all__ = ("ApplicationCommandInteraction", "ApplicationAutocompleteInteraction", "SlashOptionData")
+
+if TYPE_CHECKING:
+    from ..application_command import BaseApplicationCommand, SlashApplicationSubcommand
+    from ..state import ConnectionState
+    from ..types.interactions import Interaction as InteractionPayload
+
+
+# class is here due to circular import issues
+class SlashOptionData:
```

Comment:
Why is there documentation for this class? If it's copy-pasted here for circular import issues, then it would be internally used.

Reply:
The point behind this comment was the fact that I was originally trying to add the class to `nextcord.application_command` since there is a class of similar nature in that file: `SlashOption` (line 1218 ff.). The `SlashOptionData` class is designed to contain SlashOption data provided by a user, while the `SlashOption` class is designed to take requirements for an application command option - requirements set by the developer.

Reply:
conversation continueing in other conversation